paymentsheet-example/src/androidTest/java/com/stripe/android/lpm/TestUsBankAccountInCustomerSheet.kt

```diff
@@ -17,6 +18,7 @@ internal class TestUsBankAccountInCustomerSheet : BasePlaygroundTest() {
     ) { settings ->
         settings[CountrySettingsDefinition] = Country.US
         settings[CustomerSheetPaymentMethodModeDefinition] = PaymentMethodMode.SetupIntent
+        settings[SupportedPaymentMethodsSettingsDefinition] = "card,us_bank_account"
```

Comment:
Having to make this change seems wrong.

It seems like something went wrong when sending these up. Should we omit it, or make it null if it's empty?

Reply:
Yeah I must have tested on master by accident because that these were passing locally when ran in isolation and I though the the device is re-using settings from other tests.

It was actually the `split` call returning a list with an empty string if nothing was defined. I filtered it out now for legitimate values.